### DISCUSSION TITLE
Add mode kiosk embedded

### DISCUSSION THREAD

**Discussion by nulldoot2k** [grafana/grafana#65670] [Plugins]
now i added mode kiosk replace mode=tv
Reference: https://github.com/grafana/grafana/pull/65669

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). I am unsure what this discussion is getting across, but feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
